server/userController.js

```diff
@@ -1,55 +1,35 @@
 const redis = require('./redis.js');
 
 ////////////////////////////////////
-////////REDIS-USER FUNCTIONS
+//////// REDIS-USER FUNCTIONS
 ////////////////////////////////////
 
-function signIn (user, clientSocket) {
-
+function signIn(user, clientSocket) {
   redis.client.hgetAsync('users', user.username)
     .then(userId => {
-      if (!userId) {
-        // username does not exist
-        clientSocket.emit('signIn unsuccessful', {
+      if (userId === undefined) {
```

Comment:
Lets revisit this line to see what is being returned when the user does not exist. May be null || nil || undefined
We should adjust before forgetting!